src/stacks-hierarchy/navigation-types.ts

```diff
@@ -45,16 +45,13 @@ type PaymentParams = {
   preassignedFareProduct: PreassignedFareProduct;
 };
 
-type CarnetDetailsRouteParams = {
-  orderId: string;
-};
-
 type ReceiptScreenRouteParams = {
   orderId: string;
   orderVersion: string;
 };
 
 type FareContractDetailsRouteParams = {
+  ticketType: 'normal' | 'carnet';
```

Comment:
Should use the same `TicketType` variable as defined in `src/stacks-hierarchy/Root_FareContractDetailsScreen.tsx`

Reply:
Now that we can use the function to check if it is a carnet ticket, we should not need to pass on parameters like this, just check the `fareContract` and it just works.